# day-day-up


Recently I've found a really good way to learn python along with solving bioinformatic problems : [Rosalind Problems](http://rosalind.info/). By working out **python village** and **Bioinformatics Stronghold**, I can start accumulating little tools for future use. With **CAPS Finder** finished, I will be dedicated on solving Rosalind Challenges and update my progress in this Repository as a record for personal progress. Hope that I would become really good at Bioinformatics XD.

 #2020/01/25
 
---

Just have been learning coding for a short time.  

I created this project as a practice and meanwhile hoping to create some tools that everyone in our lab can use  in the future, hopefully :smile:.

- Up till now, I made a **PCR calculator** that calculates the amount of reagents to add into a PCR reaction, and I will give it more functions in case anyone requests. It's still not very useful right now, but I hope it will be helpful in some day.

- My next plan is to creat a tool（**CAPS Finder**） which can help you when a CAPS marker needs to be designed. With the target sequence input, it will automaticly tell you where the PCR product (or just sequence) would be cut, and print the cut sequence. Especially, in each lab, enzymes that could be used right away are limited, as other enzymes may need ordering, which costs much time. Therefore I hope this tool will help to find CAPS marker, the CAPS location, Buffer to be used and the enzymes we have at stock(currently in our lab though).

- Another plan is to use fasta data from NCBI to automaticly generate sequence with position and mark out the transcription starting point, exon ,introns and UTR. This is merely just a thought, I may work on this after **CAPS Finder** is done.


  #2020/01/15
